src/test/java/com/sparta/om/tests/PeopleDTOTests.java

```diff
@@ -92,8 +96,11 @@ void doesPreviousReturnUrl(){
         }
 
         private boolean onFirstPage() {
-            System.out.println(dto);
-            return true;
+            return page == 0;
```

Comment:
This methods shouldn't be in the tests maybe add it to the util class or add a new class entirely